aemcmc/transforms.py

```diff
@@ -0,0 +1,75 @@
+import aesara.tensor as at
+from etuples import etuple, etuplize
+from kanren import eq, lall
+from kanren.facts import Relation, fact
+from unification import var
+
+location_scale_family = Relation("location-scale-family")
+fact(location_scale_family, at.random.cauchy)
+fact(location_scale_family, at.random.gumbel)
+fact(location_scale_family, at.random.laplace)
+fact(location_scale_family, at.random.logistic)
+fact(location_scale_family, at.random.normal)
+
```

Comment:
Is this list meant to be exhaustive? If so then I'd add the uniform and t distributions as well.

Reply:
I just added the ones on the top of my head when I wrote the code, but good point

Reply:
Don't forget, a lot of these distributions are given by more general distribution(s), so we could simplify the recentering/rescaling rewrite implementations via the use of intermediate rewrites that convert them to their generalized distribution(s).